lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -143,9 +143,9 @@ <h3 class="hd hd-3 title">${_("Certificate Generation History")}</h3>
             <table>
                 <thead>
                 <tr>
-                    <td class="task-name"></td>
-                    <td class="task-date"></td>
-                    <td class="task-details"></td>
+                    <th class="task-name" scope="col">${_('Task name')}</th>
+                    <th class="task-date" scope="col">${_('Date')}</th>
```

Comment:
It was a pain in the butt to get certs running (literally took all week). The code from AC-524 is very close to this, so would [that sandbox](https://clrux-ac-524.sandbox.edx.org/courses/course-v1:edX+SOH101+2016/instructor#view-certificates) work?